NuSpecs/Humanizer.Core.de-CH.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.12">
+    <id>Humanizer.Core.de-CH</id>
+    <version>$version$</version>
+    <title>Humanizer Locale (de-CH)</title>
+    <authors>Mehdi Khalili, Oren Novotny</authors>
+    <owners>Mehdi Khalili, onovotny</owners>
+    <projectUrl>https://github.com/Humanizr/Humanizer</projectUrl>
+    <iconUrl>https://raw.github.com/Humanizr/Humanizer/master/logo.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Humanizer Locale (de-CH)</description>
+    <copyright>Copyright 2012-2015  Mehdi Khalili</copyright>
+    <licenseUrl>https://raw.githubusercontent.com/Humanizr/Humanizer/master/LICENSE</licenseUrl>
+    <language>de-CH</language>
+    <dependencies>
+       <dependency id="Humanizer.Core" version="[$version$]" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="Humanizer\bin\Release\netstandard1.0\de-CH\*.*" target="lib\netstandard1.0\de" />
```

Comment:
I think this needs to be in the `de-CH` subdirectory, see https://github.com/Humanizr/Humanizer/blob/24c3a83b744a8d1d41e0d68030ac4dbafc1bea05/NuSpecs/Humanizer.Core.bn-BD.nuspec#L21

Reply:
Yep, I overlooked that...